oauth/v1/types.go

```diff
@@ -252,3 +252,22 @@ type RedirectReference struct {
 	// The name of the target that is being referred to. e.g. name of the Route.
 	Name string `json:"name" protobuf:"bytes,3,opt,name=name"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// UserOAuthAccessToken is a virtual resource to mirror OAuthAccessTokens to
+// the user the access token was issued for
+type UserOAuthAccessToken struct {
+	OAuthAccessToken `json:",inline" protobuf:"bytes,1,opt,name=oAuthAccessToken"`
```

Comment:
I don't like this protobuf "non-inlining". Isn't there real inlining without using different protobuf ids?

Reply:
`type UserOAuthAccessToken OAuthAccessToken` could be a way

Reply:
yes

Reply:
it seems to work fine for what we're doing in https://github.com/openshift/oauth-apiserver/pull/24